<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1b1d23;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,0,10,0,2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">border-radius: 15px;
background-color: rgb(39, 44, 54);
border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,0,3,0,2,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>19</number>
          </property>
          <property name="leftMargin">
           <number>150</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="recordButton">
            <property name="font">
             <font>
              <family>-apple-system</family>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  appearance: button;
  backface-visibility: hidden;
  background-color: #405cf5;
  border-radius: 6px;
  border-width: 0;
  box-shadow: rgba(50, 50, 93, .1) 0 0 0 1px inset,rgba(50, 50, 93, .1) 0 2px 5px 0,rgba(0, 0, 0, .07) 0 1px 1px 0;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Ubuntu,sans-serif;
  font-size: 100%;
  height: 35px;
  line-height: 1.15;

  outline: none;
  overflow: hidden;
  padding: 0 5px;
  position: relative;
  text-align: center;
  text-transform: none;
  transform: translateZ(0);
  transition: all .2s,box-shadow .08s ease-in;
  touch-action: manipulation;
  width:100px;
}
QPushButton:hover{
background-color: #283999;
}
QPushButton:pressed{
background-color: #1c2973;
}
</string>
            </property>
            <property name="text">
             <string>Record</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/mic-svgrepo-com.png</normaloff>icons/mic-svgrepo-com.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="membersList">
            <property name="styleSheet">
             <string notr="true">            QListWidget {
                background-color: rgb(39, 44, 54);
                padding: 5px;
                border-radius: 15px;
                margin: 0 0 ;
				padding: 5px 0;
                font-family: Roboto-Bold;
				font-weight: 1000;
            }

            QListWidget::item {
                background-color: rgb(39, 44, 54);
                padding: 5px;
                border-bottom: 1px solid #c0c0c0;
                border-radius: 15px;
                color: #ffffff;
    
            }

            QListWidget::item:selected {
                background-color: #3399ff;
                color: #ffffff;
            }

            QListWidget::item:hover {
                background-color: #e0e0e0;
            }
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_8">
          <property name="styleSheet">
           <string notr="true">/* Line style */

  /* Set the width of the line */
  width: 20px;

  /* Set the height of the line */
  height: 5px;

  /* Set the background color of the line */
  background-color: #3a3b3c;

  /* Set the border of the line */
  border: 10px;

  /* Set the border radius of the line */
  border-radius:5px;



</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,8">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Overpass Black</family>
              <pointsize>18</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
background: rgba(74, 74, 74, 0);
margin: 0 0 ;</string>
            </property>
            <property name="text">
             <string>Spectrogram</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="spectrogram" native="true">
            <property name="styleSheet">
             <string notr="true">margin: 1px 140px;
color: white;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_9">
          <property name="styleSheet">
           <string notr="true">/* Line style */

  /* Set the width of the line */
  width: 20px;

  /* Set the height of the line */
  height: 5px;

  /* Set the background color of the line */
  background-color: #3a3b3c;

  /* Set the border of the line */
  border: 10px;

  /* Set the border radius of the line */
  border-radius:5px;



</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Overpass Black</family>
              <pointsize>18</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
background: rgba(74, 74, 74, 0);
margin: 0 0 ;</string>
            </property>
            <property name="text">
             <string>Analysis Results</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="analysisResult"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="result_label">
            <property name="font">
             <font>
              <family>Overpass Black</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
font-size:40px;
margin: 0 0 ;</string>
            </property>
            <property name="text">
             <string>Result</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">border-radius: 15px;
background-color: rgb(39, 44, 54);
border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
